.github/workflows/build.yaml

```diff
@@ -55,3 +61,36 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build-cuml:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@upkeep/manylinux-improvements
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cuml
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: cuml
+      package-dir: python
+      versioneer-override: ${{ inputs.wheel_versioneer_override }}
+
+      # Note that this approach to cloning repos obviates any modification to
+      # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
+      # the clone as is.
+      extra-repo: rapidsai/cumlprims_mg
+      extra-repo-sha: branch-23.02
```

Comment:
I think we can rely on this sha always being aligned for this workflow, right? This will always be a branch build where we want to build using the same (mainline) branch of `cumlprims_mg`?

Reply:
Not a big deal either way here, just a thought. The current approach via `update-version.sh` seems equivalent to me.

Reply:
Since this way (branch string + update-versions.sh) works across PRs, branch, and stable builds, while a different approach ("use `$get_current_branch()`") would need a special case for PRs, we'll keep it this way for now